plugins/webp-uploads/helper.php

```diff
@@ -406,3 +406,61 @@ function webp_uploads_is_picture_element_enabled(): bool {
 function webp_uploads_is_jpeg_fallback_enabled(): bool {
 	return (bool) get_option( 'perflab_generate_webp_and_jpeg' );
 }
+
+/**
+ * Retrieves the image URL for a specified MIME type from the attachment metadata.
+ *
+ * This function attempts to locate an alternate image source URL in the
+ * attachment's metadata that matches the provided MIME type.
+ *
+ * @since n.e.x.t
+ *
+ * @param int    $attachment_id The ID of the attachment.
+ * @param string $src           The original image src url.
+ * @param string $mime          A mime type we are looking to get image url.
+ * @return string|false Returns mime type image if available, false otherwise.
+ */
+function webp_uploads_get_mime_type_image( int $attachment_id, string $src, string $mime ) {
```

Comment:
This seems a bit better to return null instead of false:

```suggestion
function webp_uploads_get_mime_type_image( int $attachment_id, string $src, string $mime ): ?string {
```